tron/bin/action_runner.py

```diff
@@ -93,6 +93,24 @@ def build_environment(run_id, original_env=None):
     return new_env
 
 
+def build_labels(run_id, original_labels=None):
+    if original_labels is None:
+        original_labels = dict()
+
+    try:
+        run_num = run_id.split(".", maxsplit=3)[2]
+    except IndexError:
+        # if we can't parse the run_id, we don't want to abort, so just
+        # set these semi-arbitrarily
+        run_num = "UNKNOWN"
+
+    new_labels = dict(original_labels)
+    new_labels["TRON_RUN_NUM"] = run_num
```

Comment:
(no action needed) hopefully we can update our python version soon and use the dictionary merging operator :p